support-frontend/app/controllers/RedemptionController.scala

```diff
@@ -51,10 +50,14 @@ class RedemptionController(
   val js = "subscriptionsRedemptionPage.js"
   val css = "digitalSubscriptionCheckoutPage.css" //TODO: Don't need this?
 
-  def displayForm(redemptionCode: String): Action[AnyContent] = googleAuthAction.async {
+  def displayForm(redemptionCode: RawRedemptionCode): Action[AnyContent] = googleAuthAction.async {
```

Comment:
RawRedemptionCode is a String, it needs to be uppercased before use, probably in the next PR